### PR TITLE
docstring for observable toposeries

### PR DESCRIPTION
**Pull request by vladdez**

### CODE REVIEW

**Comment by vladdez** [comment]
- fixed #177 
- from camel to underscore functions in postion2color and getTopoPositions
- link on paper updated
- better example for topoplotscatter in docs
- dragging disanled in get_topo_positions

File: src/plot_topoplotseries.jl [hunk 1]
```diff
@@ -27,6 +28,10 @@ Multiple miniature topoplots in regular distances.
     Show labels for each electrode.
 - `positions::Vector{Point{2, Float32}} = nothing`\\
     Specify channel positions. Requires the list of x and y positions for all unique electrode.
+- `interactive_scatter = false`\\
+    Enable interactive mode. \\ 
+    If you create `obs_tuple = Observable((0, 0, 0))` and pass it into `interactive_scatter` you can change observable numbers by clicking topopplot markers.\\
```
[the remark this answers is not in the document]
Comment (behinger) [reply src/plot_topoplotseries.jl:33]: what happens if I put interactive_scatter = true without an observable?

File: src/plot_erpimage.jl [hunk 2]
```diff
@@ -52,8 +52,6 @@ plot_erpimage!(
 plot_erpimage(times::AbstractVector, data::Matrix{<:Real}; kwargs...) =
```
Comment (behinger) [inline src/plot_erpimage.jl:52]: maybe add here the Union(Observable) as well? Else you can only use observables with plot_erpimage!()

Comment (behinger) [reply src/plot_erpimage.jl:52]: I didnt see the docstring for the observables for erp_image - but maybe I missed it and they already exist. Can you confirm some user-facing documentation  exist for observables there?

File: src/plot_topoplotseries.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (vladdez) [reply src/plot_topoplotseries.jl:33]: agree, maybe `interactive_scatter = nothing` is better?

**Pull request merged**